daml-lf/spec/daml-lf-1.rst

```diff
@@ -2394,6 +2523,44 @@ exact output.
         ⇓
       Ok ⟨ f₁= v₁, …, fᵢ= vᵢ', …, fₙ= vₙ ⟩
 
+      e  ⇓  Err v
+    —————————————————————————————————————————————————————————————————————— EvExpErrorErr
+      'ERROR' @τ e  ⇓  Err v
+
+      e  ⇓  Ok v
+    —————————————————————————————————————————————————————————————————————— EvExpError
+      'ERROR' @τ e  ⇓  Err ('to_any_exception' @'UserException' ('USER_EXCEPTION' v))
+
+      e  ⇓  Err v
+    —————————————————————————————————————————————————————————————————————— EvExpThrowErr
+      'throw' @τ @σ e  ⇓  Err v
+
+      e  ⇓  Ok v
+    —————————————————————————————————————————————————————————————————————— EvExpThrow
+      'throw' @τ @σ e  ⇓  Err v
```

Comment:
This violates the invariant that in `Err v` the `v` should have `AnyException` type. But this will be fixed if we make `throw` always throw an `AnyException`, which is my suggestion.

Reply:
That's a great catch. I'll make it disappear by following your suggestion.